boards/FSM/Src/Io/Io_SteeringAngleSensor.c

```diff
@@ -0,0 +1,17 @@
+#include "Io_Adc.h"
+#include "configs/App_SteeringAngleConstants.h"
+#include "Io_SteeringAngleSensor.h"
+
+float Io_SteeringAngleSensor_GetAngleDeg(void)
+{
+    // Subtract the measured voltage by the steering angle sensor voltage
+    // offset. Multiply the result with VOLTAGE_TO_DEGREES to calculate the
+    // steering angle in degrees.
+    //
+    // The scale factor used to convert the voltage to an angle is given by 360
+    // deg / VDDA (3.3V)
+
+    const float VOLTAGE_TO_DEGREES = 360.0f / 3.3f;
+    return VOLTAGE_TO_DEGREES *
+           (Io_Adc_GetChannel1Voltage() - STEERING_ANGLE_VOLTAGE_OFFSET);
```

Comment:
`STEERING_ANGLE_VOLTAGE_OFFSET` should just be a const in this function. It does not belong in App.

Reply:
Ah okay. Ignore my comment on https://github.com/UBCFormulaElectric/Consolidated-Firmware/pull/628#discussion_r429518191

Reply:
Alright this is fixed